www/js/modules/main/directives/histogramChart.js

```diff
@@ -131,13 +131,13 @@ export default ($log) => {
         controller: histogramController,
         controllerAs: 'histogramController',
         bindToController: {
-            dataSet: '@'
+            dataset: '='
```

Comment:
intentionally lowercased this value. In html, there are special attributes called `data-` attributes. Crafting a component/directive attribute to be prefixed as `data-` is one of reasons for your issue.